dspace/config/modules/usage-statistics.cfg

```diff
@@ -35,3 +35,7 @@ usage-statistics.authorization.admin.workflow=true
 # (see query.filter.* for query filter options)
 # Default value is true.
 #usage-statistics.logBots = true
+
+# Enable/disable if a matching for a bot should be case sensitive
+# Setting this value to true will increase cpu usage, but bots will be found more accurately
+usage-statistics.bots.case-insensitive=true
```

Comment:
Per the associated ticket, DS-3558, it's mentioned in the description that this setting should be "disabled by default". Has that been rethought?